tests/system/Config/DotEnvTest.php

```diff
@@ -71,6 +72,20 @@ public function testLoadsHex2Bin()
 
 	//--------------------------------------------------------------------
 
+	public function testLoadsBase64()
+	{
+		$dotenv = new DotEnv($this->fixturesFolder, 'base64encryption.env');
+		$dotenv->load();
+
+		$value = $_ENV['encryption.key'] ?? getenv('encryption.key');
```

Comment:
Don't check `$_ENV['encryption.key']`.
```suggestion
		$value = getenv('encryption.key');
```
We have to cleanup variable `encryption.key`. Put this call before instantiating `DotEnv` object.
```php
putenv('encryption.key')
```